db/migrate/20151102113518_add_default_value_to_minimum_fixed_fee.rb

```diff
@@ -0,0 +1,12 @@
+class AddDefaultValueToMinimumFixedFee < ActiveRecord::Migration
+  def up
+    firms = Firm.where(minimum_fixed_fee: nil)
```

Comment:
No, we only care if it's in the database or not. It's just that we can't access the database if we don't have an AR model for it :)